portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -105,7 +117,7 @@ private int getIntegerParameter(HttpServletRequest request, String name, int def
   }
 
   /** Use Gson Library to convert list of comments to Json */
-  private String convertToJsonUsingGson(List<String> list) {
+  private <GenericType> String convertToJsonUsingGson(List<GenericType> list) {
```

Comment:
few nits:

* usually type parameters are represented by a single letter, i.e. `<T>`, but this is just convention to avoid confusing them with real types -- nothing wrong with what you have
* when the method implementation doesn't actually require a specific type, you can just use `<?>` instead of declaring a type parameter, e.g.

```java
private String convertToJsonUsingGson(List<?> list) {
```

Reply:
Thanks, that's exactly what I was looking for!